misk/src/main/kotlin/misk/ApplicationInterceptor.kt

```diff
@@ -0,0 +1,9 @@
+package misk
+
+interface ApplicationInterceptor {
+  fun intercept(chain: Chain): Any?
```

Comment:
I think we want to require interceptors / actions to return a non-null response (which could be ``Nothing``) or throw an exception

Reply:
+1